res/js/index.js

```diff
@@ -346,12 +348,51 @@ function handleGeolocationError(error) {
 	statusPos.html("Geolocation unavailable.");
 }
 
+var map = null;
+var markers = null;
```

Comment:
Maybe add default marker (same coordinates as placeholder: 60.1234, 5.0123) to be updated if manual input/GPSd/Geolocation available.